Remove obsolete comments

pshipton opened the pull request:
Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

DanHeidinga commented on debugtools/DDR_Autoblob/src/com/ibm/j9ddr/autoblob/GetNativeDirectory.java:
```diff
@@ -29,8 +29,6 @@
  * 
- * Based on Caroline Gough's PlatformFinder utility.
- * 
  * Andrew Hall.
```
Can you remove Andrew's name as well?

pshipton commented:
Updated

DanHeidinga commented:
Jenkins test sanity plinux

pshipton commented:
Do not merge, this breaks some DDR internals.

pshipton commented:
Apparently `Constants from <name>` is a magic DDR command

DanHeidinga commented:
Is DDR parsing the `/* Constants from ... ` comments in `j9nonbuilder.h`?

pshipton commented:
> Is DDR parsing the /* Constants from ... comments in j9nonbuilder.h?
Yes. I have restored them and will start a new (internal) build for testing.

pshipton commented:
@dmitripivkine can you please review the gc parts. I've removed some references to things that no longer exist. In particular I removed `Note: adapted from GC_ClassSuperclassesIterator in Modron/gc_structs` although there still is a gc_structs/ClassSuperclassesIterator.cpp. Maybe you want the comment fixed instead.

DanHeidinga commented:
Jenkins test sanity plinux
